app/helpers/orchid/date_helper.rb

```diff
@@ -0,0 +1,75 @@
+module Orchid::DateHelper
+
+  def clear_dates_params
+    # duplicate parameters, remove date fields, and return
+    options = params.permit!.deep_dup
+    options.delete("date_from")
+    options.delete("date_to")
+    return options
+  end
+
+  # TODO could abstract this into a CDRH gem
+  def date_default(date_params, default_date)
+    y, m, d = date_params.map { |x| x.gsub(/\s+/, "") }
+    y = default_date[0] if y.blank?
+    m = default_date[1] if m.blank?
+    d = default_date[2] if d.blank?
+
+    # solr can't handle months and days without zero padding (01)
+    m = "0#{m}" if m.length == 1
+    d = "0#{d}" if d.length == 1
+
+    return "#{y}-#{m}-#{d}"
+  end
+
+  # main method responsible for taking params and returning
+  # a set of options for querying the API
+  def date_filter aParams=params
```

Comment:
I added it add-hock because I perceived a time when we might want to pass a different object besides the params in, though looking at my code now I realize that `permit!` is going to fail on a normal hash.  Also I realize you're right, I didn't use `aParams`.  I blame my oversight on the sinus headache I've been rocking for several days, haha.

This is the situation I was picturing:
```
# in controller
options = do_something_with_text_query params
options = add_annotations_to_text options
options = date_filter options
```
or something along those lines.  I will wait to change anything to see what you think, but if we don't allow passing something else in, then we would probably need to call the date_filter method first before any other stuff in.  Or we come up with a different system to create the object being passed to the API?